### PR TITLE
fix: ranker tests

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
This Pr includes changes of https://github.com/jina-ai/jina-hub/pull/4144, so will close https://github.com/jina-ai/jina-hub/pull/4144

There's `ZEDRuntime@83[C]:can not load the executor from rankers/SimpleAggregateRanker/config.yml.` in hub-build-test ci. 
And in previous prs, and every other pr it also exists.

### CODE REVIEW

**Review by JoanFM — CHANGES_REQUESTED** [review]
The required_keys should be changed by 2 class parametrrs `query_required_keys` and `match_required_keys` set to a Tuple containing length

**Comment by Yongxuanzhang** [comment]
> The required_keys should be changed by 2 class parametrrs `query_required_keys` and `match_required_keys` set to a Tuple containing length

I made changes in the latest commit. Let me know if that works.

**Comment by JoanFM** [comment]
> This Pr includes changes of #4144, so will close #4144
> 
> There's `ZEDRuntime@83[C]:can not load the executor from rankers/SimpleAggregateRanker/config.yml.` in hub-build-test ci.
> And in previous prs, and every other pr it also exists.

This message is expected in several of our PRs

File: rankers/BiMatchRanker/__init__.py [hunk 1]
```diff
@@ -11,12 +11,13 @@ class BiMatchRanker(Chunk2DocRanker):
 
     .. warning:: Here we suppose that the smaller chunk score means the more similar.
     """
-    required_keys = {'length'}
+    query_required_keys = {'length'}
```
Comment (JoanFM) [inline rankers/BiMatchRanker/__init__.py:14]: For BiMatchRanker let it be handled in #4237

File: rankers/SimpleAggregateRanker/config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 !SimpleAggregateRanker
 with:
-  aggregate_function: min
-  is_reversed_score: True
+  aggregate_function: 'min'
```
Comment (JoanFM) [inline rankers/SimpleAggregateRanker/config.yml:3]: no need for this change

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: rankers/BiMatchRanker/__init__.py (hunk 1 above)
Comment (Yongxuanzhang) [reply rankers/BiMatchRanker/__init__.py:14]: So I will remove all changes on bimatchranker and only keep the simpleappregrate one?

**Pull request closed without merging**